server/main.lua

```diff
@@ -204,16 +190,44 @@ local function format_int(number)
     return minus .. int:reverse():gsub("^,", "") .. fraction
 end
 
+-- Get all bank cards for the current player
+local function getBankCard(cid)
+    local bankCard = MySQL.Sync.fetchAll('SELECT * FROM bank_cards WHERE citizenid = ? ORDER BY record_id DESC LIMIT 1', { cid })
```

Comment:
Use `MySQL.query.await` here

Reply:
Thanks, @BerkieBb updated to use MySQL.query.await